packages/pd-manager-rest-api/__tests__/agent.ts

```diff
@@ -0,0 +1,31 @@
+import { createAgent } from '@veramo/core'
+import { DataStore, DataStoreORM } from '@veramo/data-store'
+import { IRequiredPlugins } from '../src'
+import { DB_CONNECTION_NAME, sqliteConfig } from './database'
+import { DataSources } from '@sphereon/ssi-sdk.agent-config'
+import { KeyManager, MemoryKeyStore, MemoryPrivateKeyStore } from '@veramo/key-manager'
+import { KeyManagementSystem } from '@veramo/kms-local'
+import { PDManager } from '@sphereon/ssi-sdk.pd-manager'
+import { PDStore } from '@sphereon/ssi-sdk.data-store/dist/pd/PDStore'
+
+const dbConnection = DataSources.singleInstance().addConfig(DB_CONNECTION_NAME, sqliteConfig).getDbConnection(DB_CONNECTION_NAME)
+
+const agent = createAgent<IRequiredPlugins>({
+  plugins: [
+    new DataStore(dbConnection),
+    new DataStoreORM(dbConnection),
+    new PDManager({ store: new PDStore(dbConnection) }),
+    new KeyManager({
```

Comment:
what is this key manager doing?

Reply:
It's a copy from in contact manager and it's test code setup. This is there in other places so did not question what has to be there and what not.
Can it go out?

Reply:
Have now
> export type IRequiredPlugins = IPDManager & IDataStore
Let's see if the tests pass...